tests/simulator/test_init_from_restart.py

```
@@ -0,0 +1,84 @@
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)